otterdog/eclipse-leda.jsonnet

```diff
@@ -89,6 +89,12 @@ orgs.newOrg('eclipse-leda') {
       secret_scanning_push_protection: "disabled",
       web_commit_signoff_required: false,
     },
+    orgs.newRepo('leda-contrib-vscode-extensions') {
+      allow_update_branch: false,
+      secret_scanning: "enabled",
```

Comment:
you can remove these lines wrt secret scanning, they are already defined in the default.